Add Bypassing UDP multicast to Motionblinds (#19850)
home-assistant/home-assistant.github.io: source/_integrations/motion_blinds.markdown

For some routers "IGMP snooping" on the used wireless interface needs to be disabled to let the IGMP/multicast messages through.

For Ubiquiti routers/access points the "Enable multicast enhancement (IGMPv3)" should be disabled.

### Bypassing UDP multicast

> If UDP Multicast does not work in your setup (due to network limitations), this integration can be used in locall polling mode.

Reviewer: ```suggestion
If UDP Multicast does not work in your setup (due to network limitations), this integration can be used in local polling mode.
```

Go to Settings -> Integrations -> on the already set up Motion Blinds integration click "configure" --> disable the "Wait for push" option (disabled by default).

> The default update interval of the Motion Blinds integration is each 10 minutes, when UDP multicast pushes do not work, this polling intervall can be a bit high.

Reviewer: ```suggestion
The default update interval of the Motion Blinds integration is every 10 minutes. When UDP multicast pushes do not work, this polling interval can be a bit high.
```

> To increase the polling intervall:

Reviewer: ```suggestion
To increase the polling interval:
```

> Go to Settings -> Integrations -> on the already set up Motion Blinds integration click more options (three dots) and select "systemoptions" -> disable "polling for updates".

Reviewer: ```suggestion
Go to Settings -> Integrations -> on the already set up Motion Blinds integration click more options (three dots) and select "System options" -> disable "polling for updates".
```

> Now create an automation with as trigger a timepattern and select your desired polling time.

Reviewer: ```suggestion
Now create an automation with as trigger a time pattern and select your desired polling time.
```

> As the action select "call service" and select "update entity", select one of the motion blinds covers as entity.

Reviewer: ```suggestion
As the action select "Call service" and select "Update entity", select one of the motion blinds covers as entity.
```

For Ubiquiti routers/access points the "Enable multicast enhancement (IGMPv3)" should be disabled.

### Bypassing UDP multicast

If UDP Multicast does not work in your setup (due to network limitations), this integration can be used in local polling mode.
Go to Settings -> Integrations -> on the already set up Motion Blinds integration click "configure" --> disable the "Wait for push" option (disabled by default).

The default update interval of the Motion Blinds integration is every 10 minutes. When UDP multicast pushes do not work, this polling interval can be a bit high.
To increase the polling interval:
Go to Settings -> Integrations -> on the already set up Motion Blinds integration click more options (three dots) and select "System options" -> disable "polling for updates".
Now create an automation with as trigger a time pattern and select your desired polling time.
As the action select "Call service" and select "Update entity", select one of the motion blinds covers as entity.
You only have to create one automation with only one motion blind cover as entity, the rest will update at the same time.

> Example YAML automation for custom polling intervall (every minute):

Reviewer: ```suggestion
Example YAML automation for custom polling interval (every minute):
```
